sentinel-dashboard/src/main/webapp/resources/gulpfile.js

```diff
@@ -111,7 +111,8 @@ gulp.task('clean', function () {
 });
 
 // 总任务
-gulp.task('build', ['clean', 'jshint', 'lib', 'js', 'css']);
+// gulp.task('build', ['clean', 'jshint', 'lib', 'js', 'css']);
+gulp.task('build', ['clean', 'lib', 'js', 'css']);
```

Comment:
Why is the `jshint` task removed?

Reply:
Oh I'm sorry for commit local modify file,  I'll fix it soon.